samples/samples/snippets_test.py

```diff
@@ -816,3 +797,36 @@ def test_drop_foreign_key_contraint_delete_cascade(capsys, instance_id, sample_d
     snippets.drop_foreign_key_constraint_delete_cascade(instance_id, sample_database.database_id)
     out, _ = capsys.readouterr()
     assert "Altered ShoppingCarts table to drop FKShoppingCartsCustomerName" in out
+
+
+def test_create_sequence(capsys, instance_id, sample_database):
+    snippets.create_sequence(instance_id, sample_database.database_id)
+    out, _ = capsys.readouterr()
+    assert (
+        "Created Seq sequence and Customers table, where the key column CustomerId uses the sequence as a default value on database"
+        in out
+    )
+    assert "Number of customer records inserted is 3" in out
+    assert "Inserted customer record with Customer Id:" in out
+
+
+@pytest.mark.dependency(depends=["create_sequence"])
+def test_alter_sequence(capsys, instance_id, sample_database):
+    snippets.alter_sequence(instance_id, sample_database.database_id)
+    out, _ = capsys.readouterr()
+    assert (
+        "Altered Seq sequence to skip an inclusive range between 1000 and 5000000 on database"
+        in out
+    )
+    assert "Number of customer records inserted is 3" in out
+    assert "Inserted customer record with Customer Id:" in out
+
+
+@pytest.mark.dependency(depends=["create_sequence"])
```

Comment:
Shall we have this dependency on `alter_sequence` instead of `create_sequence`? In that way, we'll be sure that `drop_sequence` doesn't get called before `alter_sequence` is executed.